recipe/meta.yaml

```
@@ -10,16 +10,18 @@ source:
   sha256: 9775fd76d8a743d39b87df38af1cd04a58c9b5a5242d5a6350ef343d06814ab6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<3.0]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```

```suggestion
  number: 1
```
as version is not bumped, build number must

---

It's a new package so the build number `0` is correct

---

ok